src/code/z_actor.c

```diff
@@ -427,10 +427,10 @@ void Actor_GetProjectedPos(PlayState* play, Vec3f* worldPos, Vec3f* projectedPos
 }
 
 void Target_SetLockOnPos(TargetContext* targetCtx, s32 index, f32 x, f32 y, f32 z) {
```

Comment:
Looks like these have slightly different names compared to OoT. Just to make sure are you planning on renaming them as well when moving over to `Attention` (would make sense to do it then so you only have to rename the functions once)?

Reply:
Exactly. Next 1-2 PRs after this will be `Target` -> `Attention`, then after that rename, I'll finish the docs for this system. But I tried to get everything "Lock-On" in this PR, so this name can change now